packages/launcher/src/main/rpc/app.js

```diff
@@ -183,6 +186,8 @@ export const sandboxed = {
 }
 
 export const trusted = {
+  ...sandboxed,
```

Comment:
It was because the trusted UI was using the same method that was provided by sandbox but good point should keep them separate, I'll just add the same method to trusted or create a shared object.